xsl/README

```diff
@@ -1,5 +1,9 @@
 The gnucash-docs now use system xslt stylesheets and there are some customization files
-in this directory. But only images/* contents are left to make admonition and callout graphics.
+in this directory. But only images/ contents are left to make admonition and callout graphics.
+
+Additionally, the following image files of GnuCash are put into images/
+directory in convenience:
+ - gnucash-icon.png
```

Comment:
Tihis is a very old gnucash-icon, mentioned in NEWS: 1.7.7  - 05 January 2003

I see hardcoded ocurrences here:
```
./xsl/titlepage.xml:29:    <img align ="left" src="stylesheet/gnucash-icon.png" alt="GnuCash Logo"/>
./xsl/titlepage.xml:57:    <img align ="left" src="stylesheet/gnucash-icon.png" alt="GnuCash Logo"/>
./xsl/titlepage.xsl:82:    <img align="left" src="stylesheet/gnucash-icon.png" alt="GnuCash Logo"/>
./xsl/titlepage.xsl:259:    <img align="left" src="stylesheet/gnucash-icon.png" alt="GnuCash Logo"/>
```

We should replace it by one from https://github.com/gjanssens/gnucash/tree/maint/data/pixmaps.
IMHO for the titlepage gnucash_splash.png would be best.

For metadata like .destop, .rpm … we should create one based that in highcolor—perhaps overlayed by the yellow Help question mark as @gjanssens did for other components of he project.

Reply:
> We should replace it by one from https://github.com/gjanssens/gnucash/tree/maint/data/pixmaps.
> IMHO for the titlepage gnucash_splash.png would be best.

It is very difficult for me to change the layout of the titlepage, especially FO/PDF.

Reply:
Oops, I had not seen that I still was on a clone. I meant:
https://github.com/Gnucash/gnucash/tree/maint/data/pixmaps/

Reply:
Done.

> IMHO for the titlepage gnucash_splash.png would be best.

wontfix now